spec/presenters/json_job_presenter_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'spec_helper'
+
+describe JsonJobPresenter do
+
+  let(:view_context) { ActionView::Base.new }
+
+  subject(:presenter) { described_class.new(view_context) }
+
+  describe '#title' do
+    subject { presenter.title }
+
+    it { should eq '{{name}}' }
+  end
+
+  describe '#status' do
+    subject { presenter.status }
+
+    it { should eq '{{status}}' }
+  end
+
+  describe '#dom_id' do
+    subject { presenter.dom_id }
+
+    it { should eq 'job_{{id}}' }
+  end
+
+  describe '#message' do
+    its(:message) { should eq '{{#if success}}' +
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.